Makefile

```diff
@@ -0,0 +1,28 @@
+SHELL := /bin/bash
+
+.PHONY: tools lint build dependencies test
+
+tools:
+	# install gometalinter
+	go get -u github.com/alecthomas/gometalinter
+
+	# install gox
+	go get -v github.com/mitchellh/gox
+
+	# install all known linters:
+	gometalinter --install
+
+lint:
+	gometalinter --config="linters.json" ./...
+
+build:
+	@mkdir -p bin/
+	go build -o ./bin/vault-init
+	@echo
+
+	@./bin/vault-init version
```

Comment:
Continuing from https://github.com/Intellection/vault-init/pull/1/files#r332842914, this will fail because there's no `version` command so it's taken as an argument to the root command which tries to talk to Vault. In my case I don't have it running so it fails but at least it highlights the issue. See:

```console
$ make build tag=0.0.0
go build -o ./bin/vault-init

Initialising Vault...
panic: Put http://127.0.0.1:8200/v1/sys/init: dial tcp 127.0.0.1:8200: connect: connection refused

goroutine 1 [running]:
github.com/Intellection/vault-init/cmd.checkError(...)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:162
github.com/Intellection/vault-init/cmd.initVault(0x0, 0x0)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:195 +0x673
github.com/Intellection/vault-init/cmd.glob..func1(0x1e7c320, 0xc000069c40, 0x1, 0x1)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:79 +0x9b
github.com/Intellection/vault-init/vendor/github.com/spf13/cobra.(*Command).execute(0x1e7c320, 0xc00000e070, 0x1, 0x1, 0x1e7c320, 0xc00000e070)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/vendor/github.com/spf13/cobra/command.go:830 +0x2aa
github.com/Intellection/vault-init/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1e7c320, 0x103cc2a, 0x1e3e0c0, 0xc000000180)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
github.com/Intellection/vault-init/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/vendor/github.com/spf13/cobra/command.go:864
github.com/Intellection/vault-init/cmd.Execute()
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:123 +0x31
main.main()
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/main.go:26 +0x20
make: *** [build] Error 2
```

Recommended solution, just remove it and keep just the building. We usually do this just to show that the thing works.

Reply:
Removed in https://github.com/Intellection/vault-init/pull/1/commits/259f0f00ac7d7a7986c4ac8126b21023c4daaff3

```
$ make build tag=0.0.0
go build -o ./bin/vault-init


```